rosdep/python.yaml

```diff
@@ -1184,6 +1184,12 @@ python-cryptography:
   gentoo: [dev-python/cryptography]
   rhel: [python2-cryptography]
   ubuntu: [python-cryptography]
+python-cvxopt:
+  arch: [python-cvxopt]
+  debian: [python-cvxopt]
+  fedora: [python-cvxopt]
```

Comment:
I'm wondering how we want to handle python2 vs python3 in something like this.

For instance, on both Arch and Fedora, the "python-cvxopt" package actually points to the Python3 version of the package (as in both cases, Python2 is no longer shipped).  Since the semantic in rosdep for `python-*` keys seems to be "Python2", I would think we wouldn't want to add these and instead add separate `python3-*` keys.  @nuclearsandwich @tfoote thoughts?

Reply:
I didn't check for that sorry. I can add a separate "python3-cvxopt" entry for fedora and arch if it is needed.

Reply:
@clalancette On most of those platforms that have python3 by default typically the python-* rules just resolve to python3. Our recommendation is that code is written to be dual compatible. And then if you switch to python3 only you switch the keys to be python3 specific. But until then those platform users have done a great job of contributing back to package to fix any python2/3 dual compatibility bugs.